pcm_codekeys/readme.md

```diff
@@ -0,0 +1,17 @@
+# Codekeys module for Plasma Core 0.11.0
+This module enables the attachment of code to hotkeys to facilitate interactive development. 
+
+Actions are defined in `.cfg` files, supporting both pure lua and fennel syntax. prefix lua files with `ckl-` and fennel with 'ckf-' and place them in data/config. Examples of each are provided. So long as a properly named config is indexed by the engine it will be reloaded on every keypress.
```

Comment:
Every single keypress?  What about a special keypress, like F12, that means 'reload'?

Reply:
If the objective was just a reload key, then sure that'd be the way to go. The advantage of this approach is that it's easy to tweak to work around any given keybindings, and lets you bind multiple keys if desired. I've had up to four keys with different actions at some points. I don't think there's really any way to accomplish that responsively at runtime without the current structure.

Reply:
Ok.